android/app/src/test/kotlin/coop/polypoly/polypod/updatenotification/UpdateNotificationTest.kt

```diff
@@ -3,6 +3,7 @@ package coop.polypoly.polypod.updatenotification
 import androidx.test.core.app.ApplicationProvider
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.google.common.truth.Truth.assertThat
+import coop.polypoly.polypod.core.UpdateNotification
```

Comment:
This was unused? ktlint didn't say anything?

Reply:
You're such a hawk! :D ktlint did fail in the job - of course I had forgotten to manually run it :D

Reply:
No unused imports, but it did want them ordered a wee bit differently. Not that I mind, I love well ordered imports.